### DISCUSSION TITLE
Chinese translation patch for the mod?

### DISCUSSION THREAD

**Discussion by YazawaAkio** [Shpim/The-Legend-of-Bum-bo-Windfall#28] [Ideas]
I previously worked for the official Chinese translation for Legend of Bum-bo under Nicalis' request (which I'm not even sure if I've ever been credited), I was pretty thrilled upon seeing the announcement of this mod pack, so I want to see if I can help with the translation for the mod into Chinese, along with fixing the uhhh..._sheer volume_ of oversights made by the official team that I didn't get to have a say about.

**Comment by JeffAdriaanse** [comment]
I appreciate your work on the official translation! Help with translating Windfall into Chinese would be great.
